packages/plasma-new-hope/src/components/_beta/EmbeddedButton/EmbeddedButton.module.css

```
@@ -0,0 +1,170 @@
+    content: '';
+    position: absolute;
+    inset: 0;
+    z-index: 1;
+    border-radius: var(--plasma-embedded-button-border-radius);
+    outline: 0.0625rem solid transparent;
+    outline-offset: 0.125rem;
+    transition: outline-color 0.2s ease-in-out;
+    pointer-events: none;
+}
+
+.root:global(.focus-visible):focus::before,
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)